### ISSUE TITLE
Please rename "tools" folder (angular/angular/tools)

### ISSUE THREAD

**Issue by matanlurey** [dart-lang/angular_analyzer_plugin#545]
This triggers a warning every-time Angular uses `pub publish` over the single case `tool` folder.

It seems that just renaming this `plugins` would probably solve the problem.

**Comment by MichaelRFairhurst** [comment]
Yep, this should be fixed. It will need to be fixed in the analysis server, I'll file a bug there and CC brian.
